BackofficeBundle/Resources/public/coffee/table/TableviewCollectionView.coffee

```diff
@@ -4,20 +4,16 @@ search = (api, rank) ->
     return
 
 TableviewCollectionView = OrchestraView.extend(
-  events:
-    'click #none': 'clickAdd'
 
   initialize: (options) ->
+    @events = {}
+    @events['click a.ajax-add'] = 'clickAdd'
```

Comment:
You could declar the binding directly in the events array and not in the initalize method